Sources/DiffableCellKit/DifferentiableCellModelDataSource.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  DifferentiableCellModelDataSource.swift
+//  Example
+//
+//  Created by Petr Zvoníček on 08.06.18.
+//  Copyright © 2018 FUNTASTY Digital, s.r.o. All rights reserved.
+//
+
+import UIKit
+import DifferenceKit
+#if SWIFT_PACKAGE
+import CellKit
+#endif
+
+open class DifferentiableCellModelDataSource: AbstractDataSource, DataSource {
```

Comment:
The type names changed, which means all of these changes are not backward compatible. I suggest we either annotate the old types deprecated or at least think about more non-breaking solution.

Reply:
Yes, we migth include both solutions and annotate the old one. I think having those two solutions code-compatible could be dangerous and misleading. Therefore I've changed name of the type.